### PR TITLE
Service for monitoring starting and new VM-Packer

### PR DESCRIPTION
**Pull request by rasmim**
Service for monitoring was fixed;
New VM designed exclusively for monitoring created.

### CODE REVIEW

File: ubuntu-20.04SOC/scripts/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+apt update
+
+# Delete unneeded files.
+rm -f /home/vagrant/*.sh
+rm -rf /home/vagrant/netman/netman_1.0.2_linux_64-bit.zip /home/vagrant/netman/netman_1.0.2_linux_64-bit
+rm -rf /home/vagrant/gwireguard*
+rm -rf /home/vagrant/gip_*.zip
+
+# Zero out the rest of the free space using dd, then delete the written file.
+#dd if=/dev/zero of=/EMPTY bs=1M
```
Comment (rasmim) [inline ubuntu-20.04SOC/scripts/cleanup.sh:14]: Not sure what this command is doing. I receive no space left when it is not comment, at building time

Comment (mrtrkmn) [reply ubuntu-20.04SOC/scripts/cleanup.sh:14]: https://serverfault.com/questions/677021/what-is-dd-if-dev-zero-of-empty-bs-1m

Comment (mrtrkmn) [reply ubuntu-20.04SOC/scripts/cleanup.sh:14]: https://serverfault.com/questions/677021/what-is-dd-if-dev-zero-of-empty-bs-1m

**Review by rasmim — COMMENTED** [review]
Changes discussed. 

Can you approve @mrtrkmn ?

**Review by mrtrkmn — APPROVED** [review]

**Comment by ahmet** [comment]
Hello guys, I got here by you mentioning me by mistake but hey, good to meet with you keep up the good work 🤘🏼

**Pull request merged**